_monitoring-plugins/alerting/api.md

```diff
@@ -514,7 +514,7 @@ POST _plugins/_alerting/monitors
 Introduced 1.0
 {: .label .label-purple }
 
-When you update a monitor, include the current version number as a parameter. OpenSearch increments the version number automatically (see the sample response).
+When you update a monitor, you can use the `seq_no` and `primary_term` to update an existing monitor. If these numbers don’t match the existing monitor or the monitor doesn’t exist, alerting throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response).
```

Comment:
"When updating a monitor, you can include `seq_no` and `primary_term` as part of your request. If these numbers don't match the existing monitor or the monitor doesn't exist, the alerting plugin throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response)."

Reply:
Thanks for the review, I fixed the description as you mentioned.